BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -180,6 +180,37 @@ extension StackElement {
             child: child
         )
     }
+
+    /// Adds a `Spacer` element to the stack, which will grow (and shrink) depending on the amount
+    /// of left over available space when measuring the elements.
+    public mutating func addFlexibleSpacer() {
```

Comment:
Of note; this won't work with result builders. I think we may need a special element type to deal with that. Or we just make `Spacer()` have a default size of `1` so that growing and shrinking works right...